connectivity/ble_indication_hs3001/ble_app_indication_HS3001/Keil_5/ble_app_indication_HS3001.uvprojx

```diff
@@ -0,0 +1,4940 @@
+            </VariousControls>
+          </Cads>
+          <Aads>
+            <interw>1</interw>
+            <Ropi>0</Ropi>
+            <Rwpi>0</Rwpi>
+            <thumb>1</thumb>
+            <SplitLS>0</SplitLS>
+            <SwStkChk>0</SwStkChk>
+            <NoWarn>0</NoWarn>
+            <uSurpInc>0</uSurpInc>
+            <useXO>0</useXO>
+            <ClangAsOpt>4</ClangAsOpt>
+            <VariousControls>
+              <MiscControls></MiscControls>
+              <Define></Define>
+              <Undefine></Undefine>
+              <IncludePath></IncludePath>
+            </VariousControls>
+          </Aads>
+          <LDads>
+            <umfTarg>0</umfTarg>
+            <Ropi>0</Ropi>
+            <Rwpi>0</Rwpi>
+            <noStLib>0</noStLib>
+            <RepFail>1</RepFail>
+            <useFile>0</useFile>
+            <TextAddressRange>0x00000000</TextAddressRange>
+            <DataAddressRange>0x00000000</DataAddressRange>
+            <pXoBase></pXoBase>
+            <ScatterFile>.\..\src\config\copied_scatter_585_586.sct</ScatterFile>
+            <IncludeLibs></IncludeLibs>
+            <IncludeLibsPath></IncludeLibsPath>
+            <Misc> C:\Users\a5140624\Desktop\HS3001\LPCCSE-215\6.0.18.1182.1\sdk\common_project_files\misc\da14585_symbols.txt --symdefs=ble_app_peripheral_585_symdef.txt --any_placement=best_fit --datacompressor off</Misc>
```

Comment:
there is a lot of static paths in the project file. this needs to be fixed

Reply:
This has been solved. the pathes are clean now